sys/sys/fcntl.h

```diff
@@ -280,6 +280,7 @@ typedef	__pid_t		pid_t;
 #endif
 #if __BSD_VISIBLE
 #define	F_DUP2FD_CLOFORK 24		/* Like F_DUP2FD, but FD_CLOFORK is set */
```

Comment:
Why do we still need this command?

Reply:
> Why do we still need this command?

To have the close-on-fork equivalent of F_DUP2FD_CLOEXEC, as Illumos does.